### DISCUSSION TITLE
FX Middle East

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#723] [Q&A]
**Channel Name:** FX International
**Channel ID:** FXMiddleEast.us
**Country:** Middle East
**Language:** English
**Website:** xxx
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by freearhey** [comment]
According to our database this channel was closed on 1 December 2022.

https://iptv-org.github.io/channels/us/FXMiddleEast
